services/federator/test/unit/Test/Federator/Validation.hs

```diff
@@ -59,32 +59,33 @@ federateWithAllowListSuccess =
   testCase "should give True when target domain is in the list" $
     -- removing evalMock @Remote doesn't seem to work, but why?
     runM . evalMock @Remote @IO $ do
-      let allowList = RunSettings (AllowList (AllowedDomains [Domain "hello.world"]))
-      res <- Polysemy.runReader allowList $ federateWith (Domain "hello.world")
+      -- TODO: helper function for creating RunSettings
```

Comment:
This comment can be removed now

Reply:
Done in 1a449f3ff1f601f30d5555b743bd835c886ccefa